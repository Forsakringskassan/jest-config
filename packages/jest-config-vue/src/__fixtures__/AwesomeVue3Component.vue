<script lang="ts">
import { defineComponent } from "vue3";

export default defineComponent({
    name: "AwesomeVue3Component",
    props: {
        name: {
            type: String,
            required: true,
        },
    },
});
</script>

<template>
    <div>
        <p>Hello {{ name }}!</p>
    </div>
</template>
