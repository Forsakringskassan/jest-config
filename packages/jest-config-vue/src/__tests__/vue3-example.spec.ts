import { mount } from "internal-vue3-jest/test-utils";
import AwesomeVue3Component from "../__fixtures__/AwesomeVue3Component.vue";

it("vue3 should mount component", () => {
    expect.assertions(1);
    const wrapper = mount(AwesomeVue3Component, {
        propsData: {
            name: "World",
        },
    });
    expect(wrapper.html()).toMatchSnapshot();
});
