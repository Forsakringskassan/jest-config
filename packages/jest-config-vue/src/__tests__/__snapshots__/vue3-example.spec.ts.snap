// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`vue3 should mount component 1`] = `
<div>
  <p>Hello World!</p>
</div>
`;
